api/client_test.go

```diff
@@ -95,6 +96,23 @@ func TestClientToken(t *testing.T) {
 	}
 }
 
+func TestClientBadToken(t *testing.T) {
+	tokenValue := "foo\u007f"
```

Comment:
Can we also test a token that should pass the printable check?

Reply:
Oh, sure.